### PR TITLE
Init/config rollup

### PR DESCRIPTION
**Pull request by arminbro**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,5 +17,11 @@
...
     "url": "https://github.com/britannica/react-hooks/issues"
   },
-  "homepage": "https://github.com/britannica/react-hooks#readme"
+  "homepage": "https://github.com/britannica/react-hooks#readme",
+  "devDependencies": {
+    "rollup": "^1.11.3"
+  },
+  "dependencies": {
+    "react": "^16.8.6"
+  }
```
Comment (ryanwalters) [inline package.json]: `react` should be a `peerDependency`:
```js
"peerDependencies": {
  "react": ">=16.8"
}
```

File: package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
```
Comment (ryanwalters) [inline package.json]: We should point to the distributable here. After someone does `npm install @britannica/react-hooks`, this is the file that their app will look for. It's also generally recommended to use the `module` field for ES modules; but `main` should also work for us in this case. Were we to publish this in multiple formats (e.g. CommonJS as well as esm, then we would use `main` for cjs and `module` for esm).

```js
// package.json
{
  "main": "dist/bundle.js",
  "module": "dist/bundle.js"
}
```

**Comment by arminbro** [comment]
@ryanwalters made updates based on your feedback. Let me know if I missed anything. Thanks.

File: package.json [hunk 3]
```diff
@@ -2,10 +2,17 @@
...
+  "module": "dist/bundle.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c -w",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "prepublishOnly": "npm run build",
```
Comment (ryanwalters) [inline package.json]: `prepublishOnly` should be inside of `scripts`. I did a bad job mentioning that part.

```js
// package.json
{
  "scripts": {
    "prepublishOnly": "npm run build"
  }
}
```

Comment (arminbro) [reply package.json]: Okay, that makes more sense. Updated.

**Pull request merged**